### ISSUE TITLE
A11y_Visual Studio Code Web Extensions _GitHub Pull request_ Create Pull request_ Usabilty: At High contrast(Desert) mode for "Cancel,Create,Create with Option" button keyboard focus is not visible clearly.

### ISSUE THREAD

**Issue by tripathiakhil429** [microsoft/vscode-pull-request-github#5482]
**GitHub Tags:** #A11yUsable;#A11yTCS;#BM_Visual Studio Code Web Extensions_Nov2023;#Visual Studio Code Web Extensions;#ChromiumEdge;#DesktopWeb;#A11ySev4;#Win11;#Highcontrast;

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details**
Application Name: Visual Studio Code Web Extensions
URL: https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github
Operating System: Windows 11 Enterprise Insider Preview(23H2)
OS Build: 25997.1000
Edge version: 120.0.2210.7 (Official build) stable app, dev channel (64-bit)

**Pre-Requisites**
Settings<Accessibility<Contrast Themes<Desert Mode.

**Repro Steps**

1.Hit the URL: https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github and login with git hub account.
2.Tab to "More Options" button present in the left navigation panel.
3.Tab to GitHub Pull request and press enter.
4.Tab to Create Pull request and press enter
5.Tab to "Cancel, Create, Create with Option" arrow button.
6.Observe at High contrast(Desert) mode for "Cancel, Create, Create with Option" button keyboard focus is not visible clearly.

**Actual**

At High contrast(Desert) mode for "Cancel, Create, Create with Option" button keyboard focus is not visible clearly.
Note: Issue is not repro'ing with aquatic mode.

**Expected**

At High contrast(Desert) mode for "Cancel, Create, Create with Option" button keyboard focus should be visible clearly.

**User Impact**

Visually impaired users who uses high contrast for there better visibility will not be able to see the "Cancel, Create, Create with Option" button keyboard focus is not visible clearly.

**Attachment**

[5482 A11y_Visual Studio Code Web Extensions _GitHub_ Description_ keyboard At High contrast(Desert) mode for Cancel,Create,Create with Option button keyboard focus is not visible clearly.webm](https://github.com/microsoft/vscode-pull-request-github/assets/85679252/ffbc2254-5985-4153-8462-14dcfeacaf8f)
